wee_slack.py

```diff
@@ -2400,6 +2401,11 @@ def resolve_ref(ref):
         # Something else, just return as-is
     return ref
 
+def create_user_status_string(profile):
+    status_emoji = profile.get("status_emoji") if profile.get("status_emoji") else "None"
+    status_text = profile.get("status_text") if profile.get("status_text") else "None"
+    return "[{}] {}".format(status_emoji, status_text)
```

Comment:
I have a freshish suggestion. Make the topic:
`username [status_emoji] "status"`
if the status emoji or status are unset, just don't render them and leave it the username.